Source/WebCore/page/Page.cpp

```diff
@@ -398,6 +399,7 @@ Page::Page(PageConfiguration&& pageConfiguration)
 #endif
 #if ENABLE(WEB_AUTHN)
     , m_authenticatorCoordinator(makeUniqueRef<AuthenticatorCoordinator>(WTFMove(pageConfiguration.authenticatorCoordinatorClient)))
+    , m_credentialRequestCoordinator(makeUniqueRef<CredentialRequestCoordinator>(WTFMove(pageConfiguration.credentialRequestCoordinatorClient)))
```

Comment:
This was the cause of the assertions.  CredentialRequestCoordinator is RefCounted, so we can't have a UniqueRef/unique_ptr to it managing its lifetime.

Reply:
Yes, there's actually no reason for it to be ref counted. It's only ever owned uniquely.